src/components/Files/FilesTab.tsx

```diff
@@ -586,60 +597,105 @@ const FileUploadDialog = ({
           </DialogTitle>
         </DialogHeader>
         <div className="space-y-4">
-          {fileUpload.files.map((file, index) => (
-            <div key={index} className="space-y-2">
-              <div className="flex items-center justify-between gap-2 rounded-md bg-secondary-300 px-4 py-2">
-                <span className="flex items-center truncate">
-                  <CareIcon icon="l-paperclip" className="mr-2 shrink-0" />
-                  <span className="truncate max-w-xs" title={file.name}>
-                    {file.name}
-                  </span>
-                </span>
-                <Button
-                  variant="ghost"
-                  size="icon"
-                  onClick={() => fileUpload.removeFile(index)}
-                  disabled={fileUpload.uploading}
-                >
-                  <CareIcon icon="l-times" />
-                </Button>
-              </div>
+          {isPdf ? (
+            <>
               <div className="space-y-2">
+                <ul className="list-disc list-inside space-y-1">
```

Comment:
Add the paperclip + background to maintain consistent behavior. Do a self-review, check against existing behavior:

<img width="1428" alt="image" src="https://github.com/user-attachments/assets/5adb4561-7b52-4a1f-9c15-a64cf7828ca1" />